<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow { background-color: #FFFFFF; }</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QHBoxLayout" name="_3">
       <property name="topMargin">
        <number>8</number>
       </property>
       <property name="bottomMargin">
        <number>8</number>
       </property>
       <item alignment="Qt::AlignLeft">
        <widget class="QWidget" name="widget" native="true">
         <layout class="QHBoxLayout" name="_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="addVideosButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Add Videos</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="clearButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Clear</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>clear-black-18dp.svg</normaloff>clear-black-18dp.svg</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="findIntrosButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Find Intros</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>search-black-18dp.svg</normaloff>search-black-18dp.svg</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item alignment="Qt::AlignRight">
        <widget class="QWidget" name="widget_3" native="true">
         <layout class="QHBoxLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="uncategorizedLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="toolTip">
             <string>Filter uncategorized</string>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgba(0, 0, 0, 0.6);
border-bottom-style: dotted;
border-radius: 0px;
border-bottom-width: 1px;
border-bottom-color: rgba(0, 0, 0, 0);</string>
            </property>
            <property name="text">
             <string>Uncategorized</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="goodFitLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="toolTip">
             <string>Filter videos with &gt;= 80% intro match</string>
            </property>
            <property name="styleSheet">
             <string notr="true">color: #007e33;
border-bottom-style: dotted;
border-radius: 0px;
border-bottom-width: 1px;
border-bottom-color: rgba(0, 0, 0, 0);</string>
            </property>
            <property name="text">
             <string>Good Fit</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="badFitLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="toolTip">
             <string>Filter videos with &lt; 80% intro match</string>
            </property>
            <property name="styleSheet">
             <string notr="true">color: #cc0000;
border-bottom-style: dotted;
border-radius: 0px;
border-bottom-width: 1px;
border-bottom-color: rgba(0, 0, 0, 0);</string>
            </property>
            <property name="text">
             <string>Bad Fit</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="selectAllButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Select All</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="deselectAllButton">
            <property name="text">
             <string>Deselect All</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>1</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>1</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">#line { 
	background-color: rgba(0, 0, 0, 0.12);
}</string>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <widget class="QWidget" name="videoFilesContainer">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1000</width>
         <height>596</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">#videoFilesContainer {
	background-color: #FAFAFA;
}</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>11</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>11</number>
        </property>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="progressBarContainer" native="true">
      <layout class="QVBoxLayout" name="_4">
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="enabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open..</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
